velox/substrait/SubstraitToVeloxExpr.h

```diff
@@ -63,6 +64,22 @@ class SubstraitVeloxExprConverter {
       const ::substrait::Expression::IfThen& substraitIfThen,
       const RowTypePtr& inputType);
 
+  /// Convert Substrait SingularOrList into Velox Expression.
+  core::TypedExprPtr toVeloxExpr(
+      const ::substrait::Expression::SingularOrList& singularOrList,
+      const RowTypePtr& inputType);
+
+  /// Create a constant expression from constant expressions by wrapping into
+  /// array.
+  std::shared_ptr<const core::ConstantTypedExpr> toConstantExpr(
+      const std::vector<std::shared_ptr<const core::ConstantTypedExpr>>&
+          constantExprs);
+
+  /// Return the function prefix.
+  std::string getFuncPrefix() {
```

Comment:
use const& for return value and make this method const

Reply:
Fixed.